app/views/govuk_publishing_components/components/_add_another.html.erb

```diff
@@ -0,0 +1,10 @@
+<%
+  add_button_text ||= "Add another"
+%>
+
+<div data-module="add-another" class="gem-c-add-another">
+  <div class="gem-c-add-another__repeated-fields js-add-another__repeated-fields">
```

Comment:
I would have liked to have used the fieldset component here, but it doesn't implement the component wrapper so I can't add the JS hook class to it. Do you think I should add the wrapper to the fieldset class?